setup.cfg

```diff
@@ -42,7 +42,8 @@ classifiers =
 # packages it depends on, namespaces, and so forth.
 
 [options]
-#install_requires =
+install_requires =
+    requests
```

Comment:
you should add the version number.

Reply:
Yep! This helps avoid the [dependency confusion vulnerability](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610)